user-guide/Advanced_Functionality/Security/Advanced_security_configuration/Security_NATS.md

```diff
@@ -0,0 +1,61 @@
+---
+uid: Security_NATS
+---
+
+# Securing NATS
+
+## Enabling TLS in a NATS cluster
+
+By default, NATS does **not** employ TLS encryption, leaving communication susceptible to eavesdropping. Consequently, we **strongly recommend enabling TLS encryption** for enhanced security within your NATS cluster.
+
+> [!NOTE]
+> This applies solely to instances involving a DMS cluster or external DxMs. If you have a single-Agent setup, all communication is confined to the local host, as there is only one NATS node.
+
+To enable authentication, please apply the following procedure to all the NATS nodes in the cluster:
+
+1. Request or generate a TLS certificate (in PEM format).
+
+   To generate self-signed certificates, we recommend that you **use our [scripts for generating TLS certificates](https://github.com/SkylineCommunications/generate-tls-certificates)**, available on GitHub. There is a version of the script for Linux and for Windows machines. The script requires two tools: *openssl* and the *Java keytool*. Both of these can run on Linux and Windows.
```

Comment:
The Linux indication is present because the script does not need to run in the machine where DataMiner is installed, meaning that the user can decide to run it on Linux and then copy over the generated certificates to the Windows machine.